gatsby-template/gatsby-config.js

```diff
@@ -12,8 +12,8 @@ const configuration = {
   bio: `{{ bio }}`,
   author: '{{ authorName }}',
   githubUrl: '{{ githubURL }}',
-  // replace this by the url where your website will be published
-  siteUrl: 'http://localhost:8000',
+  // fallback option if NetlifyURL isnt provided
+  siteUrl: '{{ siteURL }}' || 'http://localhost:8000',
```

Comment:
The fallback option isn't going to do anything since `{{ siteURL }}` isn't empty. Instead the fallback should be specified in the code.